packages/@pollyjs/adapter-node-http/src/-private/http-wrapper.js

```diff
@@ -0,0 +1,38 @@
+import http from 'http';
+import https from 'https';
+
+import semver from 'semver';
+
+import TransportWrapper from './transport-wrapper';
+
+export default class HttpWrapper {
+  constructor({ adapter, options }) {
+    this.transports = [];
+
+    if (options.transports.includes('http')) {
+      this.transports.push(
+        new TransportWrapper(http, { name: 'http', adapter })
+      );
+    }
+
+    if (
+      options.transports.includes('https') &&
+      // Node 8 and below use http.request under the hood for https.request
+      // so we should skip wrapping it even if enabled.
+      // https://github.com/nodejs/node/blob/v8.14.0/lib/https.js#L245
+      semver.gte(process.version, '9.0.0')
```

Comment:
@jasonmit @yasinuslu FYI. Im not sure if we should just throw here and force the user to to remove `https` from the transports list instead. I opted for a silent skip since this will be removed once we not longer support Node 6 & 8.

Reply:
I think you're right about silently skipping, maybe we can add it to the readme as a warning

Reply:
Agree with @yasinuslu, seems fine just needs documented.